driver/linuxtv/Makefile

```diff
@@ -64,8 +64,7 @@ else
 # builds can be reproducible
 PKG_VERS      = 0fe857b
 PKG_GIT_HASH  = 0fe857b86addf382f6fd383948bd7736a3201403
-MEDIA_VERSION = 2022-06-05-f2906aa86338
-#MEDIA_VERSION = LATEST
+MEDIA_VERSION = LATEST
```

Comment:
Issue in using this is that the builds become no longer reproducible.  That's why specific build version is being used, in hope to update to newer builds once every little while.

Reply:
I think the archive was removed  and the download broken so I used latest instead